README.md

```diff
@@ -209,6 +265,7 @@ information about the governance of the io.js project, see
 * **Colin Ihrig** ([@cjihrig](https://github.com/cjihrig)) &lt;cjihrig@gmail.com&gt; (Technical Committee)
 * **Mikeal Rogers** ([@mikeal](https://github.com/mikeal)) &lt;mikeal.rogers@gmail.com&gt;
 * **Rod Vagg** ([@rvagg](https://github.com/rvagg)) &lt;rod@vagg.org&gt;
+<br>Release GPG key: DD8F2338BAE7501E3DD5AC78C273792F7D83545D
```

Comment:
Maybe a naive question but can't we use a single master key with as many sub-keys as there are release managers?  It would make all releases be signed by "io.js Release Engineering &lt;releng@iojs.org&gt;" (or whatever we decide to call ourselves) and it makes revocation a little easier.  Master key management is an open issue, though; who gets to own it?